src/main/java/org/folio/inventory/dataimport/handlers/actions/AbstractAuthorityEventHandler.java

```diff
@@ -64,6 +64,7 @@ protected AbstractAuthorityEventHandler(Storage storage, MappingMetadataCache ma
 
   @Override
   public CompletableFuture<DataImportEventPayload> handle(DataImportEventPayload payload) {
+    LOGGER.trace("handle:: parameter payload: {}", payload);
```

Comment:
Does it makes sense to add LOGGER.debug("handle:: parameter payload is: {}", payload == null ? "null" : ''not null");

Reply:
Hello guys, we don't need here check for payload ==null, because if paylod is null, this method/handler will not be called